src/debug.c

```diff
@@ -551,11 +551,9 @@ NULL
         emptyDb(-1,EMPTYDB_NO_FLAGS,NULL);
         protectClient(c);
         int ret = loadAppendOnlyFile(server.aof_filename);
+        if (ret == AOF_CRIT || ret == AOF_OPEN_ERR)
```

Comment:
maybe a white-list of errors is better than a black list.
i.e. explicitly ignore OK and EMPTY

Reply:
relpaced